test/Utils/UtilsParser.test.js

```diff
@@ -62,4 +62,59 @@ describe('#utilsParser', () => {
         });
     });
   });
+  describe('referenceDataReq()', () => {
+    it('should parse sample data response for Country', () => {
+      const uParser = new Parser('util:ReferenceDataRetrieveRsp', 'v47_0', {});
+      const parseFunction = utilsParser.REFERENCE_DATATYPE;
+      const xml = fs.readFileSync(`${xmlFolder}/country-data.xml`).toString();
+      return uParser.parse(xml).then((json) => {
+        const result = parseFunction.call(uParser, json);
+        expect(result).to.be.an('array').and.to.have.lengthOf(3);
+        result.forEach(c => expect(c).to.be.an('object').and.to.have.all.keys(['Code', 'Name', 'Deprecated']));
```

Comment:
Nice one! Could we transform keys to camelCase as well? So resulting object would have next shape:

```
type Response = ReferenceData[];

type ReferenceData = {
  name: string,
  code: string,
  deprecated: boolean
}
```

Reply:
Parser updated to dynamically rewrite the keys to camelCase